### PR TITLE
Create a lightweight version of cloud-deps containers

### PR DESCRIPTION
**Pull request by grahamplata**
Minimal (default): `rill devtool start cloud` → Starts core services only.
Full:` rill devtool start cloud --full` → Starts core + extras services.
Other presets (e.g., e2e, other) remain unchanged

**Checklist:**
- [ ] Covered by tests
- [x] Ran it and it works as intended
- [x] Reviewed the diff before requesting a review
- [x] Checked for unhandled edge cases
- [x] [Linked](https://linear.app/rilldata/issue/INFR-126/create-a-lightweight-version-of-cloud-depsdocker-composeyaml) the issues it closes
- [ ] Checked if the docs need to be updated. If so, create a separate Linear DOCS issue
- [ ] Intend to cherry-pick into the release branch
- [ ] I'm proud of this work!

### CODE REVIEW

File: cli/cmd/devtool/start.go [hunk 1]
```diff
@@ -252,6 +262,14 @@ func (s cloud) start(ctx context.Context, ch *cmdutil.Helper, verbose, reset, re
 		return fmt.Errorf("error parsing .env: %w", err)
 	}
 
+	if !full {
+		os.Setenv("RILL_ADMIN_METRICS_EXPORTER", "prometheus")
```
Comment (ericpgreen2) [inline cli/cmd/devtool/start.go]: What's the reasoning for this? Prometheus is excluded from the "minimal" config.

File: cli/cmd/devtool/start.go [hunk 2]
```diff
@@ -252,6 +262,14 @@ func (s cloud) start(ctx context.Context, ch *cmdutil.Helper, verbose, reset, re
...
+		os.Setenv("RILL_ADMIN_TRACES_EXPORTER", "")
+		os.Setenv("RILL_RUNTIME_METRICS_EXPORTER", "prometheus")
```
Comment (ericpgreen2) [inline cli/cmd/devtool/start.go]: Same question here

File: cli/cmd/devtool/start.go (hunk 1 above)
Comment (grahamplata) [reply cli/cmd/devtool/start.go]: The application sends metrics to zipkin when in local. When the container is't present the application will complain.

Comment (ericpgreen2) [reply cli/cmd/devtool/start.go]: I think the correct way to fix this would be to not emit metrics when the environment variable isn't set.

Comment (grahamplata) [reply cli/cmd/devtool/start.go]: Sure, let me address that

Comment (grahamplata) [reply cli/cmd/devtool/start.go]: Originally they were always set in the various `.envs` so the cleanest way would be to set them when using full. Short term I updated the needful

**Review by ericpgreen2 — APPROVED** [review]
LGTM. Thanks for doing this!
